README.md

```diff
@@ -9,78 +9,39 @@
 [![Website](https://img.shields.io/website?url=https%3A%2F%2Foqt.ohsome.org)](https://oqt.ohsome.org)
 [![status: active](https://github.com/GIScience/badges/raw/master/status/active.svg)](https://github.com/GIScience/badges#active)
 
-
-## Vision
-
-What is OQT?
-* A project that **formalizes the knowledge on OSM data quality** within HeiGIT and the GIScience Research Group.
-* A tool that end users, e.g. humanitarian organisations and public administration, can use to get information on the **quality of OSM data for their specific region and use-case**.
-* A **web app** that builds upon the existing infrastructure, especially [ohsomeHeX](https://ohsome.org/apps/osm-history-explorer) and the [ohsome dashboard](https://ohsome.org/apps/dashboard).
-* A **data integration tool**, that brings together the implementation of a variety of intrinsic and extrinsic data quality metrics.
```

Comment:
Should this be in the README?

Reply:
This is also on Confluence.

Reply:
Hey,
I think we do not need to state it as the vision, but for sure need 2-3 general sentences what OQT is about. But what you have below might be already enough.

so yes, let's remove it from the readme.